src/rules/interfaceOverTypeLiteralRule.ts

```diff
@@ -57,6 +57,17 @@ function walk(ctx: Lint.WalkContext): void {
             if (ctx.sourceFile.text[node.end - 1] === ";") {
                 fix.push(Lint.Replacement.deleteText(node.end - 1, 1));
             }
+            // replace trailing comma with semicolon
+            const members = node.type.members;
+            if (members.length !== 0) {
+                members.forEach(mem => {
+                    const memText: string = mem.getFullText();
+                    if (memText[memText.length - 1] === ",") {
```

Comment:
If you fork this rule and use it in your own project, you might want to write some test cases around comments and spacing:

```ts
type Wat = {
    someMember : string /* gotcha! */,
}
```

Reply:
Got it. Thanks!